scene/2d/collision_object_2d.cpp

```diff
@@ -82,7 +82,7 @@ uint32_t CollisionObject2D::create_shape_owner(Object *p_owner) {
 	uint32_t id;
 
 	if (shapes.size() == 0) {
-		id = 1;
+		id = 0;
 	} else {
 		id = shapes.back()->key() + 1;
```

Comment:
`+ 1` here should be removed?

Reply:
I don't think so, isn't removing the `+1` will make the id same as the last one? I've compiled the source and test the `create_shape_owner` 2 times, the result is right `[0, 1]`. Or am I missing something?

Reply:
Oh, okay. I misunderstood the logic. 😝